### DISCUSSION TITLE
no db apps install

### DISCUSSION THREAD

**Discussion by ersingo** [caprover/caprover#1758] [General]
Hello friends;

I want to install the application of the files. I created a database. The pictures are below. I don't know if I did it right. There's a mistake somewhere. I can't install the application. It gives an error in the last stage. Is my job correct? I'm not a master. I am inexperienced. I set up on the VPS server to my own domain.

DB Create
https://ibb.co/bgDG26m

Result
https://ibb.co/dJYYj5w

App config
https://ibb.co/w62Rb3C

Details
https://ibb.co/QmgpYNR

Setup screen
https://ibb.co/WGHFspn

Error
https://ibb.co/7vXfzXt

**Comment by githubsaturn** [comment]
One click apps are community maintained. Please comment on the PR that last touched this definition:
https://github.com/caprover/one-click-apps/pull/653
